src/metabase/db/custom_migrations.clj

```diff
@@ -42,7 +42,8 @@
   `(defrecord ~name []
      CustomTaskChange
      (execute [_# database#]
-       ~migration-body)
+       (t2/with-transaction [_conn#]
```

Comment:
run each custom migrations in a transaction to prevent it from commit prematurelly if error happens